# HG changeset patch
# User "Benoît Thébaudeau" <benoit.thebaudeau@advansee.com>
# Date 1323692879 -3600
# Node ID 8e6fc0fe3bc9e1fbab213fc869aeb910a7da0588
# Parent  e98c6a720ce858a254ef3dbd4e5db5e245797706
debug/gdb: update Linaro GDB revisions to 2011.12

Update Linaro GDB with the latest available revisions.

Signed-off-by: "Benoît Thébaudeau" <benoit.thebaudeau@advansee.com>

diff --git a/config/debug/gdb.in b/config/debug/gdb.in
--- a/config/debug/gdb.in
+++ b/config/debug/gdb.in
@@ -34,9 +34,9 @@
 # Don't remove next line
 # CT_INSERT_VERSION_BELOW
 
-config GDB_V_linaro_7_3_2011_10
+config GDB_V_linaro_7_3_2011_12
     bool
-    prompt "linaro-7.3-2011.10 (EXPERIMENTAL)"
+    prompt "linaro-7.3-2011.12 (EXPERIMENTAL)"
     depends on DEBUG_GDB_SHOW_LINARO
     select GDB_7_0_or_later
 
@@ -97,7 +97,7 @@
     string
 # Don't remove next line
 # CT_INSERT_VERSION_STRING_BELOW
-    default "linaro-7.3-2011.10" if GDB_V_linaro_7_3_2011_10
+    default "linaro-7.3-2011.12" if GDB_V_linaro_7_3_2011_12
     default "7.3a" if GDB_V_7_3a
     default "linaro-7.2-2011.05-0" if GDB_V_linaro_7_2_2011_05_0
     default "7.2a" if GDB_V_7_2a
